packages/cfd/src/Containers/cfd-dashboard.tsx

```diff
@@ -401,14 +405,16 @@ const CFDDashboard = (props: TCFDDashboardProps) => {
 
     const is_suspended_mt5_demo_server = getIsSuspendedMt5Server(mt5_status_server.demo);
     const is_suspended_mt5_real_server = getIsSuspendedMt5Server(mt5_status_server.real);
+    const is_suspended_dxtrade_demo_server = !!dxtrade_status_server.demo;
+    const is_suspended_dxtrade_real_server = !!dxtrade_status_server.real;
 
     const has_mt5_account_error = is_demo_tab
         ? is_suspended_mt5_demo_server || has_mt5_demo_account_error || mt5_disabled_signup_types.demo
         : is_suspended_mt5_real_server || has_mt5_real_account_error || mt5_disabled_signup_types.real;
 
     const has_dxtrade_account_error = is_demo_tab
-        ? has_dxtrade_demo_account_error || dxtrade_disabled_signup_types.demo
-        : has_dxtrade_real_account_error || dxtrade_disabled_signup_types.real;
+        ? is_suspended_dxtrade_demo_server || has_dxtrade_demo_account_error || dxtrade_disabled_signup_types.demo
+        : is_suspended_dxtrade_real_server || has_dxtrade_real_account_error || dxtrade_disabled_signup_types.real;
```

Comment:
Can you put these conditions inside circular brackets both truthy and falsy , It would be better for readability